src/app/routes/utils/constructPageFetchUrl/index.ts

```diff
@@ -96,6 +97,20 @@ const getId = ({ pageType, service, variant, env }: GetIdProps) => {
       getIdFunction = () => pageType;
       break;
     case LIVE_PAGE:
+      getIdFunction = (path: string) => {
+        const isTipoId = isTipoIdCheck(path);
+        const isCpsId = isCpsIdCheck(path);
+        const id = isTipoId ? getTipoId(path) : getCpsId(path);
```

Comment:
think we can remove this line to make it easier to read / remove one conditional.

e.g. 



```
if (isTipoId) {
  return getTipoId(path);
}
if (isCpsId) {
  return `/${service}/live/${getCpsId(path)}`;
}

```

Reply:
Yep, so think it's:

```
    case LIVE_PAGE:
      getIdFunction = (path: string) => {
        if (isTipoIdCheck(path)) {
          return getTipoId(path);
        }
        if (isCpsIdCheck(path)) {
          return `/${service}/live/${getCpsId(path)}`;
        }
        return null;
      };
      break;
```